src/util/atomic.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+// Qemu faults if you try load or store with a strict memorder on aarch64.
+#ifdef BOARD_qemu_arm_virt
```

Comment:
Just making a note here so I don't forget that we should return to do this. I'll chuck it in a GitHub issue soon.